spec/models/book_spec.rb

```diff
@@ -0,0 +1,65 @@
+require 'rails_helper'
+
+RSpec.describe Book, type: :model do
+  subject(:book) do
+    Book.new(
+      genre: genre, author: author, image: image, title: title,
+      editor: editor, year: year
+    )
+  end
+
+  let(:genre)            { Faker::Book.genre }
+  let(:author)           { Faker::Book.author }
+  let(:title)            { Faker::Book.title }
+  let(:editor)           { Faker::Book.publisher }
+  let(:image)            { Faker::Name.last_name }
+  let(:year)             { Faker::Number.number(4) }
+
+  describe '#create' do
+    context 'When the genre is nil' do
+      let(:genre) { nil }
+
+      it do
+        is_expected.to be_invalid
```

Comment:
We should take advantage of [shoulda_matchers](https://github.com/thoughtbot/shoulda-matchers) validators! Using it we recude this to only 1 line.